src/Components/NavBar/NavBarLoggedInActions.tsx

```diff
@@ -123,6 +133,14 @@ export const NavBarLoggedInActionsQueryRenderer: React.FC<{}> = () => {
       environment={relayEnvironment}
       query={graphql`
         query NavBarLoggedInActionsQuery {
+          notificationsConnection(first: 1) {
```

Comment:
**question**: maybe it's worth query more than 1 artworks here? so how do we then do some data filtering?

Reply:
increased to 3, thanks!